subsys/nrf_rpc/rpc_utils/Kconfig

```diff
@@ -31,7 +31,36 @@ config NRF_RPC_DEV_INFO_CLIENT
 
 endchoice
 
-config DEV_INFO_RPC_INITIALIZE_NRF_RPC
+menuconfig NRF_RPC_CRASH_GEN
+	bool "nRF RPC crash generator"
+	depends on NRF_RPC
+	depends on NRF_RPC_CBOR
+	help
+	  nRF RPC crash generating functionalities
+
+if NRF_RPC_CRASH_GEN
```

Comment:
Should it be inside `if NRF_RPC_DEV_INFO`? In CMakeLists it seems you wanted to make both configs independent of each other.

Reply:
yes, these should be separated, fixed